fission/main.go

```diff
@@ -103,6 +103,7 @@ func main() {
 	fnLogCountFlag := cli.StringFlag{Name: "recordcount", Usage: "the n most recent log records"}
 	fnForceFlag := cli.BoolFlag{Name: "force", Usage: "Force update a package even if it is used by one or more functions"}
 	fnExecutorTypeFlag := cli.StringFlag{Name: "executortype", Usage: "Executor type for execution; one of 'poolmgr', 'newdeploy' defaults to 'poolmgr'"}
+	fnTestDebugFlag := cli.BoolFlag{Name: "debug", Usage: "return internal error details as part of http response for debugging"}
```

Comment:
If you have a per-function "dev/production" flag, you can remove the header and this flag, and just unconditionally return errors in dev mode

Reply:
@soamvasani, just my opinion on this, I like this per-request level of debugging; it makes debugging a one-of request quick and easy. Use cases: demos, explanations, and troubleshooting a production-level function (you don't want all workload to suddenly be in debug mode). 

Isn't updating the function spec before and after a request to set and unset debug mode a bit verbose from a user's perspective, or am I missing a faster workflow to do this?